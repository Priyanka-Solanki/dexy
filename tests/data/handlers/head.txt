* Lorem ipsum dolor sit amet, consectetur adipiscing elit.
* Pellentesque lobortis augue in ipsum tristique a rhoncus eros dapibus.
* Vivamus a elit ut dolor scelerisque cursus ut sit amet turpis.
* Integer euismod nulla id urna consequat aliquam.
* Donec ac est semper lectus sagittis posuere non vitae velit.
* Nullam posuere orci vitae tortor lacinia sagittis at a neque.

* Vivamus porttitor erat sit amet lectus semper quis auctor dui porttitor.
* Quisque at enim diam, vel malesuada mauris.
* Integer semper ante lacus, quis hendrerit augue.
* Vivamus congue rhoncus arcu, vitae mattis enim hendrerit ac.
* Nullam aliquet mi at nulla dapibus sodales.
* Maecenas porttitor est eu diam condimentum nec posuere odio iaculis.
* Phasellus vel elit malesuada mi feugiat tempus pulvinar sed ipsum.

* Proin non enim vitae nisi dapibus hendrerit quis et est.
* Sed commodo enim quis felis mattis sit amet eleifend arcu pretium.
* Mauris ut enim vitae sapien vestibulum laoreet a consequat mauris.
* Nullam sit amet felis id mauris rhoncus ultricies.
* Proin dignissim odio volutpat arcu luctus laoreet.
* Cras at leo in odio posuere hendrerit nec vel orci.
* Donec at lectus in nunc lacinia pretium.
