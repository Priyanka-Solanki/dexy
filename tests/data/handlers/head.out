* Lorem ipsum dolor sit amet, consectetur adipiscing elit.
* Pellentesque lobortis augue in ipsum tristique a rhoncus eros dapibus.
* Vivamus a elit ut dolor scelerisque cursus ut sit amet turpis.
* Integer euismod nulla id urna consequat aliquam.
* Donec ac est semper lectus sagittis posuere non vitae velit.
* Nullam posuere orci vitae tortor lacinia sagittis at a neque.

* Vivamus porttitor erat sit amet lectus semper quis auctor dui porttitor.
* Quisque at enim diam, vel malesuada mauris.
* Integer semper ante lacus, quis hendrerit augue.
